docs/rst/formalia/titlepage.rst

```diff
@@ -11,8 +11,19 @@
   :alt: eProsima
   :target: http://www.eprosima.com/
 
-*eProsima DDS Visualizer Plugin* is a graphical desktop application providing real-time 2D charts to visualize
+|ddsvisualizer| is a graphical desktop application providing real-time 2D charts to visualize
 topic-related data traffic information from *eProsima Fast DDS* entities.
+This application provides visualizations of time series and X-Y plots with topic-related DDS data, showing the
+user a quick and rich representation of the detailed data content that is being exchanged in the network.
```

Comment:
Please, add a bit more information about the plugin.
* Explain the use case where this will be useful
* Comment about the amount of features to plot data (visualize time series, modify axes and layouts, create custom series from ones existing, etc.)
* Add requirements/features as lists